packages/react-components/src/components/abstract-xy-output-component.tsx

```diff
@@ -269,12 +271,15 @@ export abstract class AbstractXYOutputComponent<P extends AbstractOutputProps, S
                     outputStatus: treeResponse.status,
                     xyTree: treeResponse.model.entries,
                     columns
+                }, () => {
+                    this.recursiveFetch();
```

Comment:
see related comment https://github.com/eclipse-cdt-cloud/theia-trace-extension/pull/865#issuecomment-1380572897

Reply:
I don't how it happened, but when testing it the while loop in `recursiveFetch()` never finishes. Maybe `props.experiment` is never updated?

Reply:
I couldn't find a case where recursiveFetch fails to finish, but removed it as requested in https://github.com/eclipse-cdt-cloud/theia-trace-extension/pull/865#issuecomment-1380572897